app/src/main/java/com/sunlightlabs/android/congress/fragments/CommitteeListFragment.java

```diff
@@ -196,7 +197,7 @@ else if (context.type == COMMITTEE)
 		private List<Committee> forLegislator(String bioguideId) {
 			List<Committee> committees;
 			try {
-				committees = CommitteeService.forLegislator(bioguideId);
+				committees = LegislatorService.find(bioguideId).committees;
```

Comment:
Parsing committee memberships for a legislator now comes from a legislator-oriented endpoint, not a committee-oriented endpoint, so the parsing code has moved from one service to the other.